frontend/src/app/firewood/forests/order-confirmation/order-confirmation.component.html

```diff
@@ -19,18 +19,16 @@ <h2>Required documents</h2>
         <li>Your load tags</li>
         <li>Cutting map</li>
       </ul>
-      <input id="includeRules" type="checkbox" name="includeRules" [checked]="includeRules" (change)="includeRules = !includeRules">
-      <label for="includeRules">Include a copy of the firewood cutting regulations</label>
-      <button id="tree-print-permit" class="usa-button-primary usa-button-big" type="button" [disabled]="isPermitExpired" (click)="printPermit()">
+      <button id="tree-print-permit" class="usa-button-primary usa-button-big" type="button" [disabled]="isPermitExpired" (click)="printPDF()">
         Print now
       </button>
-      <p>These documents have also been emailed to <strong>email@address.com</strong></p>
+      <p>These documents have also been emailed to <strong>{{permit.emailAddress}}</strong></p>
     </div>
 
     <div *ngIf="forest" id='tree-order-summary-container' class="confirmation-details-container clear margin-top-2">
       <div *ngIf="isPermitExpired" class="usa-alert usa-alert-error permit-expiration-warning">
         <div class="usa-alert-body">
-          <!-- <h3 class="usa-alert-heading">This permit was issued on {{permit.transactionDate | date: 'shortDate'}} and expired on {{permit.expirationDate | date: 'shortDate'}}</h3> -->
```

Comment:
Same as the prior comments about this being addressed in another issue. Please omit all the changes in this block that are out of scope for 1478.